_std/defines/component_defines/component_defines_global.dm

```diff
@@ -36,3 +36,28 @@
 /// ⠀⠀⠀⠀⠀⠀⠀⠈⠛⠻⠿⠿⠿⠿⠋⠁⠀⠀⠀⠀
 #define COMSIG_GLOBAL_SUSSY_PHRASE "sussy"
 #define COMSIG_GLOBAL_UNCOOL_PHRASE "uncool_word"
+
+// ---- Generic Dock Events ----
+
+	/// Shuttle is about to arrive at a dock
+	#define COMSIG_DOCK_EVENT_INCOMING "dock_incoming"
+
+	/// Shuttle has arrived
+	#define COMSIG_DOCK_EVENT_ARRIVED "dock_arrived"
+
+	/// Shuttle is about to depart
+	#define COMSIG_DOCK_EVENT_OUTGOING "dock_outgoing"
+
+	/// Shuttle has departed from the dock
+	#define COMSIG_DOCK_EVENT_DEPARTED "dock_departed"
```

Comment:
these appear to be being used as arguments *for* a signal, rather than a signal themselves, and so shouldn't be using the `COMSIG_` naming convention

Reply:
moved to `component_defines_special.dm` and added comments there describing the pattern.